Transcom/TrainScheduleBuilder.cs

```diff
@@ -0,0 +1,42 @@
+﻿using AutoMapper;
+using PSITranscom.Models;
+using System.Collections.Generic;
+using System.Linq;
+using Transcom.Dtos;
+
+namespace Transcom
+{
+    public class TrainScheduleBuilder : ITrainScheduleBuilder
+    {
+        private readonly IMapper _mapper;
+
+        public TrainScheduleBuilder(IMapper mapper)
+        {
+            this._mapper = mapper;   
+        }
+        public List<TrainScheduleDto> BuildTrainSchedule(
+            List<Schedule> schedules,
+            List<DailyRoute> dailyRoute,
+            List<Timetable> timetables
+            )
+        {
+
+            var trainSchedules = new List<TrainScheduleDto>();
+
+            foreach (var schedule in schedules)
+            {
+                var selectedTimetable = timetables.Where(s => s.TrainNumber == schedule.TrainNumber).OrderBy(s => s.SequenceNumber).ToList();
+                var selectedDailyRoute = dailyRoute.Where(rd => rd.RunningDayCode == schedule.RunningCode).ToList();
```

Comment:
`selectedDailyRoute` is not used. This means you're not using the `dailyRoute` parameter either. Do you need them at all?